ArcGISRuntimeSDKQt_CppSamples/Analysis/StatisticalQueryGroupSort/StatisticalQueryGroupSort.cpp

```diff
@@ -84,8 +84,11 @@ void StatisticalQueryGroupSort::connectSignals()
     addResultToModel("", QString("Error. %1").arg(error.message()));
   });
 
-  connect(m_featureTable, &ServiceFeatureTable::queryStatisticsCompleted, this, [this](QUuid, StatisticsQueryResult* result)
+  connect(m_featureTable, &ServiceFeatureTable::queryStatisticsCompleted, this, [this](QUuid, StatisticsQueryResult* rawResult)
   {
+    // Delete rawResult when we leave local scope.
+    QScopedPointer<StatisticsQueryResult> result(rawResult);
```

Comment:
Do we need to include this header? Same question anywhere it's used.

Reply:
Header now included where used